_unittest/test_28_Maxwell3D.py

```diff
@@ -793,3 +794,44 @@ def test_48_simplify_objects(self):
         assert not self.aedtapp.simplify_objects(
             input_objects_list="impedance_box", simplify_type="Polygon Fit", extrusion_axis="U"
         )
+
+    def test_49_cylindrical_gap(self):
+        example_project = os.path.join(local_path, "example_models", test_subfolder, cyl_gap + ".aedt")
+        m3d = Maxwell3d(example_project, specified_version=desktop_version)
+        [
+            x.delete()
+            for x in m3d.mesh.meshoperations[:]
+            if x.type == "Cylindrical Gap Based" or x.type == "CylindricalGap"
+        ]
+        assert m3d.mesh.assign_cylindrical_gap("Band", meshop_name="cyl_gap_test")
+        assert not m3d.mesh.assign_cylindrical_gap(["Band", "Inner_Band"])
+        assert not m3d.mesh.assign_cylindrical_gap("Band")
+        [
+            x.delete()
+            for x in m3d.mesh.meshoperations[:]
+            if x.type == "Cylindrical Gap Based" or x.type == "CylindricalGap"
+        ]
+        assert m3d.mesh.assign_cylindrical_gap(
+            "Band", meshop_name="cyl_gap_test", clone_mesh=True, band_mapping_angle=1
+        )
+        [
+            x.delete()
+            for x in m3d.mesh.meshoperations[:]
```

Comment:
Out of curiosity, why do we need to handle the slices here with `[:]` to delete some mesh operations?